src/main/java/frc/robot/subsystems/elevator/ElevatorConstants.java

```diff
@@ -9,5 +9,7 @@ public class ElevatorConstants {
     public final static double KP = 0;
     public final static double KI = 0;
     public final static double KD = 0;
+
+    public final static double SPEED_LIMIT = 0;
```

Comment:
change that to the real value you want. I think we want 1 to be the limit (this is the maximum of the motor)